src/components/Loading/Loading.js

```diff
@@ -4,7 +4,7 @@ import { caribbeanGreen } from 'utils/colors'
 
 export default function Loading ({
   color = caribbeanGreen,
-  type = 'TailSpin',
+  type = 'Circles',
```

Comment:
did Zoe or C sign off on this change? When C and I worked on the loaders, she specifically didn't like this one

Reply:
lol. That was me playing around a bit/testing out options to talk about later down the road. Did not mean to leave that in there. I already changed it back.